openapi.yaml

```diff
@@ -8862,9 +9410,10 @@ paths:
       servers:
       - url: https://api.linode.com/v4beta
       summary: Create an Object Storage Key
-      description: >
+      description: |
```

Comment:
what do `>` and `|` mean here?

Reply:
The | respects line breaks without having to use markup where the > does not.

Reply:
This is actually not accurate, it's a yaml scalar block thing: https://yaml-multiline.info